Makefile.kokkos

```diff
@@ -279,6 +284,18 @@ else
   KOKKOS_INTERNAL_OPENMPTARGET_FLAG := -fopenmp -foffload=nvptx-none
 endif
 
+# Set OpenACC flags.
+ifeq ($(KOKKOS_INTERNAL_COMPILER_PGI), 1)
+  KOKKOS_INTERNAL_OPENACC_FLAG := -acc
+else
+    ifeq ($(KOKKOS_INTERNAL_COMPILER_NVHPC), 1)
+      KOKKOS_INTERNAL_OPENACC_FLAG := -acc
```

Comment:
if these two do the same thing why not have them in the same group?

Reply:
I removed the KOKKOS_INTERNAL_COMPILER_PGI lines.
In fact NVHPC and PGI is the same, I removed that in one of the previous comments that I forgot to remove this too.